modules/ROOT/pages/install-rtfctl.adoc

```diff
@@ -52,70 +52,534 @@ curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -
 
 [%header%autowidth.spread,cols="a,a"]
-.rtfctl Flags
 |===
-|Flag |Description
-| -h, --help | Displays `rtfctl` help information.
-| -n, --namespace string | If present, displays the namespace scope.
-| --pod string | If present, displays the pod name.
+|Command |Description
+|xref:install[`install`] | Installs Runtime Fabric
+|xref:reinstall[`reinstall`] | Re-installs Runtime Fabric components
+|xref:upgrade-cluster[`upgrade-cluster`] | Upgrades the Runtime Fabric cluster
+|xref:backup[`backup`] | Performs a back up of the local Runtime Fabric state
+|xref:restore[`restore`] | Restores local Runtime Fabric state from a backup
+|xref:validate[`validate`] | Validates the cluster for Runtime Fabric installation
+|xref:version[`version`] | Displays the Runtime Fabric version information
 |===
 
-== Usage
+=== `install`
+Installs Runtime Fabric
 
-* To list all supported commands, run `rtfctl -h`. 
+----
+rtfctl install <activation data>
+----
 
-* For more information about a specific command, run `rtfctl <command> -h`.
+=== `reinstall`
+
+Re-installs Runtime Fabric components
+
+* Re-installs Runtime Fabric components using the latest scripts.
++
+----
+rtfctl reinstall
+----
+
+* Re-installs Runtime Fabric components without downloading the latest scripts.
++
+----
+rtfctl reinstall --skip-download
+----
+
+=== `backup`
+Performs a back up of the local Runtime Fabric state. 
+----
+rtfctl backup /var/backups/<backup-filename>.tar.gz
+----
+
+=== `restore`
+Restores local Runtime Fabric state from a backup 
+
+----
+rtfctl restore /var/backups/<filename>.tar.gz
+----
+
+=== `upgrade-cluster` 
+Upgrades the Runtime Fabric cluster
+
+* Upgrades the cluster using a pre-downloaded package
++
+----
+rtfctl upgrade-cluster --file runtimefabric-<version>.tar.gz
+----
+
+* Upgrades the cluster by downloading a package
++
+----
+rtfctl upgrade-cluster --url https://<host>/<path-to-installer-package>
+----
+
+=== `validate`
+Validates the cluster for Runtime Fabric installation
+
+----
+rtfctl validate <activation-data>
+----
 
-=== Install a Mule license:
+=== `version` 
+Displays the Runtime Fabric version information
 
-.. Make sure to base64 encode the license, following the instructions in xref:install-manual#base64-encode-your-mule-license-key[Base64 Encode your Mule License Key].
-.. Run the following command:
+----
+rtfctl version
+----
+
+== Commands for Managing Applications on Runtime Fabric
+
+Use the following commands to manage applications deployed to Runtime Fabric.
+
+[%header%autowidth.spread,cols="a,a"]
+|===
+|Command |Description
+|xref:delete[`delete`] | Deletes a secure property
+|xref:describe[`describe`] | Displays details for an application
+|xref:disk[`disk`] | Displays disk usage for an application
+|xref:memory[`memory`] | Displays the JVM memory usage in a pod
+|xref:package[`package`] | Creates a debugging information package for an application
+|xref:restart[`restart`] | Restarts an application
+|===
+
+=== `delete`
```

Comment:
This should probably be in the same group as `apply secure-property`

Reply:
I think in the case of this structure it doesn't make sense to put them together. And we do have a separate page for secure properties.